angular/projects/spark-angular/src/lib/components/sprk-promo/sprk-promo.stories.ts

```diff
@@ -35,18 +35,41 @@ const modules = {
 export const defaultStory = () => ({
   moduleMetadata: modules,
   template: `
-    <sprk-promo
-      title="Title"
-      subtitle="Subtitle"
-      cta="button"
-      buttonHref="#"
-      ctaText="Learn More"
-      hasBorder="true"
-      idString="promo-1"
-    >
-      Lorem ipsum dolor. Sit amet pede. Tempus donec et. Suspendisse id
-      inventore integer eum non enim diam habitant. Maecenas nunc per lacus
-      neque egestas. Diam quod curabitur.
+    <sprk-promo idString="default-promo" hasBorder="true">
+      <div sprkStackItem sprkPromoContent>
+        <sprk-stack itemSpacing="large">
+          <h3
+            sprkHeading
+            variant="displayFive"
+            sprkPromoHeading
+            sprkStackItem
+          >
+            Title
+          </h3>
+
+          <p sprkStackItem sprkPromoSubheading sprkText variant="bodyOne">
+            Subtitle
+          </p>
+
+          <p sprkText variant="bodyTwo" sprkStackItem>
+            Lorem ipsum dolor. Sit amet pede. Tempus donec et. Suspendisse id
+            inventore integer eum non enim diam habitant. Maecenas nunc per lacus
+            neque egestas. Diam quod curabitur.
+          </p>
+
+          <div sprkStackItem>
+            <a
+              sprkLink
+              variant="unstyled"
+              href="#nogo"
```

Comment:
User can now use routerLink in promo here by updating this to have the routerLink directive instead of standard href.